core/lib/zksync_core/src/api_server/web3/state.rs

```diff
@@ -543,10 +546,53 @@ impl<E> RpcState<E> {
 /// Contains mapping from index to `Filter` with optional location.
 #[derive(Default, Debug, Clone)]
 pub struct Filters {
-    state: HashMap<U256, TypedFilter>,
+    state: HashMap<U256, InstalledFilter>,
     max_cap: usize,
 }
 
+#[derive(Debug, Clone)]
+struct InstalledFilter {
+    pub filter: TypedFilter,
+    #[allow(dead_code)]
```

Comment:
This lint is usually resolved by naming a field with a `_` prefix, e.g. `_guard`.